### PR TITLE
[5.1] Make ThrottleLogins Configurable

### PR DESCRIPTION
**Pull request by mul14**
We can change the throtlle "maximum attempts" and "delay time" by edit `config/auth.php` file.

> This PR depends on https://github.com/laravel/laravel/pull/3443

### CODE REVIEW

File: src/Illuminate/Foundation/Auth/ThrottlesLogins.php [hunk 1]
```diff
@@ -17,8 +17,8 @@ protected function hasTooManyLoginAttempts(Request $request)
     {
         $attempts = $this->getLoginAttempts($request);
 
-        if ($attempts > 3) {
-            Cache::add($timeKey = $this->getLoginLockExpirationKey($request), time() + 60, 1);
+        if ($attempts > config('auth.throtlle.maximum')) {
```
Comment (lukasgeiter) [inline src/Illuminate/Foundation/Auth/ThrottlesLogins.php]: Typo. `throttle` not `throtlle`

File: src/Illuminate/Foundation/Auth/ThrottlesLogins.php [hunk 2]
```diff
@@ -17,8 +17,8 @@ protected function hasTooManyLoginAttempts(Request $request)
...
+            Cache::add($timeKey = $this->getLoginLockExpirationKey($request), time() + config('auth.throtlle.delay'), 1);
```
Comment (lukasgeiter) [inline src/Illuminate/Foundation/Auth/ThrottlesLogins.php]: Typo. `throttle` not `throtlle`

**Comment by mul14** [comment]
@lukasgeiter Thanks, I'll send new commit.

**Comment by Arrilot** [comment]
You need to pass 3 and 60 as second parameters to `config()`. These values aren't set on any existing L5.1 install.

**Comment by GrahamCampbell** [comment]
This needs rebasing.

**Comment by JesseLeite** [comment]
If user doesn't pull in fresh instance of Laravel, they won't have these settings in /config/auth.php.  I think if user doesn't have this set in config, you should still default to a hard-coded integer.  I would suggest something like 5, not 3 (my reasoning is explained in #9504).

**Comment by taylorotwell** [comment]
I don't really plan to make this configurable, at least not via a config file.

**Comment by JesseLeite** [comment]
Fair enough.  I've bugged you too much about this already, but maybe consider PR #9504 @taylorotwell?

**Pull request closed without merging**